### PR TITLE
Build flavour changes

### PR DESCRIPTION
**Pull request by svillar**
A couple of changes:
* rename x86_64 to x64 for simplicity
* enforce API level 29 for AOSP to support MagicLeap2 (the recent upgrade to API 33 broke it)

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -645,7 +646,7 @@ dependencies {
     geckoImplementation deps.gecko_view."${branch}_arm64"
     configurations.all {
          resolutionStrategy.capabilitiesResolution.withCapability('org.mozilla.geckoview:geckoview') {
-            def abi = getName().toLowerCase().contains('x86_64') ? 'x86_64' : 'arm64'
+            def abi = getName().toLowerCase().contains('x64') ? 'x86_64' : 'arm64'
```
Comment (javifernandez) [inline app/build.gradle:649]: s/x86_64/x64 ?

Comment (svillar) [reply app/build.gradle:649]: nop, because this is used to get the candidate name for the geckoview build, and it uses x86_64

**Comment by elima** [comment]
> rename x86_64 to x64 for simplicity

Isn't x86 a better term? It speaks to the underlying architecture and it is kind of well known, whereas x64 is not immediately associated with x86_64, so requires extra cognition effort.

**Comment by svillar** [comment]
> > rename x86_64 to x64 for simplicity
> 
> Isn't x86 a better term? It speaks to the underlying architecture and it is kind of well known, whereas x64 is not immediately associated with x86_64, so requires extra cognition effort.

The thing is that x86 is generally used for 32bit architectures. x64 is a [well-known variant](https://en.wikipedia.org/wiki/X86-64) of x86_64

**Review by javifernandez — APPROVED** [review]

**Pull request merged**